### ISSUE TITLE
t/01-repo.t fails on Darwin

### ISSUE THREAD

**Issue by eserte** [jacquesg/p5-Git-Raw#179]
With 0.66 and 0.67 I see the following test failure with self-compiled perls (5.12.3, 5.24.0, 5.25.8):
```
#   Failed test at t/01-repo.t line 79.
#          got: '0'
#     expected: '1'
# Looks like you failed 1 test of 125.
t/01-repo.t ................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/125 subtests 
```

Looking at the failing test line the reason is clear:
```
is $caps{ignore_case}, 1 if ($^O eq 'darwin' || $^O eq 'MSWin32');
```
I configured my filesystem to be case-sensitive.

**Comment by eserte** [comment]
To find out if the file system is case sensitive one can use the following command:
```
$ diskutil info / | grep File.System
   File System Personality:  Case-sensitive Journaled HFS+
```

**Comment by jacquesg** [comment]
I've just ignored the problem :) db59ae3520f8cac1b388dddf0892bef3910a475b

**Comment by jacquesg** [comment]
Scrap that, fixed in 9a45d36c8fbb02524997df1ca9569376b73343f0

**Comment by jacquesg** [comment]
Fixed, the change will go into the next release. Thanks again for reporting this.
